src/lib/arrays.js

```diff
@@ -0,0 +1,123 @@
+const getNthElement = (index, array) => {
+  const index2 = index - array.length;
+
+  if (index >= array.length) {
+    return array[index2];
+  }
+  return array[index];
+};
+
+const arrayToCSVString = array => {
+  return array.toString();
+};
+
+const csvStringToArray = string => {
+  return string.split(',');
+};
+
+const addToArray = (element, array) => {
+  array.push(element);
+};
+
+const addToArray2 = (element, array) => {
+  return array.concat([element]);
+};
+
+const removeNthElement = (index, array) => {
+  return array.splice(index, 1);
+};
+
+const numbersToStrings = numbers => {
+  const numberString = numbers.map(number => {
```

Comment:
You dont use the value numberString anywhere else apart from the return, therefor you could just return it straightaway without being assigned:
````
return numbers.map(number => {
    return number.toString();
  });
`````